lib/github_classroom/lti/membership_service.rb

```diff
@@ -20,25 +20,42 @@ def instructors(body_params: nil)
         membership(roles: %w[Instructor], body_params: body_params)
       end
 
-      def membership(roles: [], body_params: nil)
-        req = membership_request(roles, body_params)
-        response = send_request(req)
+      def membership(membership_url: @context_membership_url, roles: [], body_params: nil)
+        total_membership = []
+
+        body = fetch_raw_membership(membership_url, roles, body_params)
+        total_membership += parse_membership(body)
+
+        # pagination
+        if @lti_version == 1.1
```

Comment:
Does the pagination only exist for 1.1?

Reply:
Yup! Version 1.0 doesn't support that capability.

Reply:
Not sure if we need to check the version since we're still handling the situation where `nextPage` doesn't exist.